src/code/z_demo.c

```diff
@@ -795,109 +824,112 @@ void Cutscene_Command_GiveTatlToPlayer(PlayState* play, CutsceneContext* csCtx,
+                    play->envCtx.screenFillColor[3] = 255.0f * lerp;
                 } else {
-                    play->envCtx.screenFillColor[3] = (1.0f - temp_f0) * 255.0f;
+                    play->envCtx.screenFillColor[3] = (1.0f - lerp) * 255.0f;
                 }
                 break;
 
-            case 3:
-            case 7:
+            case CS_TRANS_RED_FILL_OUT:
+            case CS_TRANS_RED_FILL_IN:
                 play->envCtx.screenFillColor[0] = 255;
                 play->envCtx.screenFillColor[1] = 0;
                 play->envCtx.screenFillColor[2] = 0;
-                if (cmd->base == 3) {
-                    play->envCtx.screenFillColor[3] = (1.0f - temp_f0) * 255.0f;
+                if (cmd->type == CS_TRANS_RED_FILL_OUT) {
+                    play->envCtx.screenFillColor[3] = (1.0f - lerp) * 255.0f;
                 } else {
-                    play->envCtx.screenFillColor[3] = 255.0f * temp_f0;
+                    play->envCtx.screenFillColor[3] = 255.0f * lerp;
                 }
                 break;
 
-            case 4:
-            case 8:
+            case CS_TRANS_GREEN_FILL_OUT:
+            case CS_TRANS_GREEN_FILL_IN:
                 play->envCtx.screenFillColor[0] = 0;
                 play->envCtx.screenFillColor[1] = 255;
                 play->envCtx.screenFillColor[2] = 0;
-                if (cmd->base == 4) {
-                    play->envCtx.screenFillColor[3] = (1.0f - temp_f0) * 255.0f;
+                if (cmd->type == CS_TRANS_GREEN_FILL_OUT) {
+                    play->envCtx.screenFillColor[3] = (1.0f - lerp) * 255.0f;
                 } else {
-                    play->envCtx.screenFillColor[3] = 255.0f * temp_f0;
+                    play->envCtx.screenFillColor[3] = 255.0f * lerp;
                 }
                 break;
 
-            case 9:
+            case CS_TRANS_TRIGGER_INSTANCE:
                 gSaveContext.cutsceneTransitionControl = 1;
                 break;
 
-            case 10:
-            case 11:
+            case CS_TRANS_BLACK_FILL_OUT:
+            case CS_TRANS_BLACK_FILL_IN:
                 play->envCtx.screenFillColor[0] = 0;
                 play->envCtx.screenFillColor[1] = 0;
                 play->envCtx.screenFillColor[2] = 0;
-                if (cmd->base == 10) {
-                    play->envCtx.screenFillColor[3] = (1.0f - temp_f0) * 255.0f;
+                if (cmd->type == CS_TRANS_BLACK_FILL_OUT) {
+                    play->envCtx.screenFillColor[3] = (1.0f - lerp) * 255.0f;
                 } else {
-                    play->envCtx.screenFillColor[3] = 255.0f * temp_f0;
+                    play->envCtx.screenFillColor[3] = 255.0f * lerp;
                 }
                 break;
 
-            case 12:
-                play->envCtx.screenFillColor[0] = (160.0f * (1.0f - temp_f0));
+            case CS_TRANS_BLACK_C:
+                play->envCtx.screenFillColor[0] = 160.0f * (1.0f - lerp);
```

Comment:
```suggestion
            case CS_TRANS_BLACK_C:
                play->envCtx.screenFillColor[0] = (1.0f - lerp) * 160.0f;
```
To match the other formats.
Out of curiosity have you tried this or `CS_TRANS_BLACK_D`? Based on the code it kind of looks like a change from an all gray screen to an all black screen and vice versa.

Reply:
Yeah, that's what it does look like. No, I haven't tried it. I'll try and record it later today and post it in discord